### ISSUE TITLE
remove UUID from database export

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#414]
The UUID that's attached to the exported database file during an export is unnecessary and makes the file less readable.

**Comment by vadekh** [comment]
@dwhieb would that just entail removing the UUID part of this line?
https://github.com/digitallinguistics/app/blob/3f5912c1fd15b5b58c41197442b50f18f5974456/src/app/App.js#L391

**Comment by dwhieb** [comment]
Yes, although now that I look at it, those four lines of code are repeated in App.js and Settings.js. Settings.js could just make a call to App.exportData() instead to make the code DRYer. I don't remember any particular reason for needing that code repeated... but if something breaks I guess we'll find out!

`App` is in the global scope, so Settings.js can call it directly.
